notebooks/3-Model-pipeline.ipynb

```diff
@@ -49,7 +49,7 @@
     }
```

Comment:
<p>The point of overfitting is less clear on this example because the results on training and test set are quite similar. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/esciencecenter-digital-skills/SICSS-odissei-machine-learning/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>